### PR TITLE
Milestone 5

### PR DESCRIPTION
**Pull request by Ceejayski**
In this milestone, I implemented the styling of the different components based on the design given by Zeplin.

### CODE REVIEW

File: src/components/Book/index.js [hunk 1]
```diff
@@ -1,20 +1,63 @@
...
+  //   <td>
+  //     { book.title }
+  //   </td>
+  //   <td>
+  //     { book.category }
+  //   </td>
+  //   <td>
+  //     <button onClick={() => handleRemoveBook(book)} type="submit">Remove</button>
+  //   </td>
+  // </tr>
```
Comment (rukundoeric) [inline src/components/Book/index.js]: - [ ] <b>REQUIRED:</b> To make sure that your codebase is always clean, remember to remove commented-codes.

File: src/components/Navbar/style.css [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+.BOOKS {
+  width: 59px;
+  margin: 0 41px 11px 0;
+}
+
+.CATEGORIES {
+  width: 105px;
+  margin: 0 664px 11px 41px;
+  opacity: 0.5;
+}
```
Comment (rukundoeric) [inline src/components/Navbar/style.css]: - [ ] <b>REQUIRED:</b> There are still few things we need to fix to make our app look more like the provided design. Kindly review the following issues and make sure to fix them.

| Original      | Yours |
  | ----------- | ----------- |
| 80047697-0c3f6280-8506-11ea-9416-a4f1800e2329 | scrnli_10_8_2021_4-57-51 PM |
   - The app should not be fullscreen. It should be like a card in the center of the page as it is in the mockup design.
   - The categories selector should be in the toolbar as it is in the mockup design
   - There are should be a line separating the booklist and form

**Review by rukundoeric — CHANGES_REQUESTED** [review]
Hi @Ceejayski, 

_Great job so far 👏 💯 👍. However, there are few issues we need to address before we move on. Please, check the comments below._
## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 

## Required Changes ♻️
  - [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
  - [ ] Check the comments under the review.

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

File: src/components/Navbar/style.css [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+.Mask {
+  width: 17px;
+  height: 17px;
+  color: #0290ff;
+  font-size: 17px;
+  display: block;
+  margin: auto;
+}
```
Comment (rukundoeric) [inline src/components/Navbar/style.css]: - [ ] <b>REQUIRED:</b> There are still few things we need to fix to make our app look more like the provided design. Kindly review the following issues and make sure to fix them.

| Original      | Yours |
  | ----------- | ----------- |
| 80047697-0c3f6280-8506-11ea-9416-a4f1800e2329 | scrnli_10_8_2021_8-13-23 PM |
   - As mentioned in the previous review, The app should not be fullscreen. It should be like a card in the center of the page as it is in the mockup design.
   - As mentioned in the previous review, The categories selector should be in the toolbar as it is in the mockup design
   - And There are should be a line separating the booklist and form

**Review by rukundoeric — CHANGES_REQUESTED** [review]
Hi @Ceejayski, 

_Great job so far 👏 💯 👍. You've done well implementing some of the requested changes. However, there are still some which aren't addressed yet. Please, check the comments below._
## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 
  - Addressed some of the issues raised from the previous review. 💯 

## Required Changes ♻️
  - [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
  
  - N/A 

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/components/Navbar/style.css (hunk 3 above)
Comment (Ceejayski) [reply src/components/Navbar/style.css]: Sorry having checked other projects like this, I have seen that this was meant to be a full-screened website and not the  card-like website as ask on the first review

**Review by eliasecasta — APPROVED** [review]
## Status:  Approved 🟢

Hi, @Ceejayski, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**